src/map-custom-fields.js

```diff
@@ -197,16 +197,16 @@ export default (function MapCustomFields () {
     }, result)
   }
 
-  function mapNumber (rawNo, regEx = CONSTANTS.field.integer) {
+  function mapNumber (rawNo) {
     const result = {}
     if (!isValidValue(rawNo))
       return result
-    const matchedNumber = regEx.exec(rawNo)
-    if (!matchedNumber) {
+    const isNotValidNumber = isNaN(Number(rawNo))
+    if (isNotValidNumber) {
       result.error = `The number ${rawNo} isn't valid`
       return result
     }
-    result.data = parseInt(matchedNumber[0], 10)
+    result.data = parseFloat(rawNo, 10)
```

Comment:
I'm wondering if it's not safer to store the number parsed with `Number` to test against **and** return. Because now it parses the number in two different ways which might lead unexpected behaviour.